static/js/jira-connect-ghe.js

```diff
@@ -0,0 +1,63 @@
+const ALLOWED_PROTOCOLS = ["http:", "https:"];
+const GITHUB_CLOUD = ["github.com", "www.github.com"];
```

Comment:
Is this is the check for users who are meaning to connect to cloud but accidentally go down the server path? Shouldn't we do a hard check against the github.com url? https://github.com. My thoughts are it would need to be an exact match for us to assume this is what we are trying to do, otherwise, we make a request to what they enter and render the error when it fails. Plus, the first item 'github.com' could cause problems for users who have internal instance running on a domain that is something like 'http://my-internal-github.com'

Reply:
Yes this simply checks if the hostname is one of the `GITHUB_CLOUD`. I am doing a `===` check, so it is doing an exact match with the user's input URL.  
I am doing a `hostname` check, so shouldn't be a problem for domain names like `http://my-internal-github.com`.